catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -135,92 +121,13 @@ public void setStreamCreationPlugin(StreamCreationPlugin streamCreationPlugin) {
     }
 
     public Double getDistanceTolerance() {
-        return this.distanceTolerance;
+        return context.getGeometryOperatorContext()
+                .getDistanceTolerance();
     }
 
     public void setDistanceTolerance(Double distanceTolerance) {
```

Comment:
Nulls are OK. I annotated the param with @Nullable, as well as the getter getDistanceTolerance.